packages/wrangler/src/deployment-bundle/create-worker-upload-form.ts

```diff
@@ -556,7 +580,14 @@ export function createWorkerUploadForm(worker: CfWorkerInit): FormData {
 		...(tail_consumers && { tail_consumers }),
 		...(limits && { limits }),
 		...(annotations && { annotations }),
-		...(experimental_assets && { assets: experimental_assets.jwt }),
+		...(experimental_assets && {
+			assets: {
+				jwt: experimental_assets.jwt,
+				...Object.fromEntries(
+					Object.entries(assetConfig).filter(([k, v]) => v !== undefined)
```

Comment:
Can't see suggestions in the UI for some reason...

I'm seeing a lint error on the `k`. Should be `_`.
```javascript
Object.entries(assetConfig).filter(([_, v]) => v !== undefined)
```